src/app/components/turnos/gestor-agendas/gestor-agendas.scss

```
@@ -18,3 +18,23 @@ span.badge,
 span.badge-default {
     font-size: 0.6rem;
 }
+table{
+    height:90%; 
```

```suggestion
    height: 100%; 
```
con 90% queda un margen blanco demasiado grande :see_no_evil: